src/config.js

```diff
@@ -0,0 +1,26 @@
+/*global System */
+'use strict';
+
+System.config({
+    transpiler: "babel",
+    packages: {
+         './': { defaultExtension: false },
+        // './_actions': { main: 'index.js' },
```

Comment:
delete commented code from your codebase. If code isn't being used, don't let it litter the codebase